samples/apps/spring-petclinic/.patch/application-postgresql.properties.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/src/main/resources/application-postgres.properties b/src/main/resources/application-postgres.properties
+deleted file mode 100644
+index 80bd3e1..0000000
+--- a/src/main/resources/application-postgres.properties
++++ /dev/null
+@@ -1,6 +0,0 @@
+-database=postgres
+-spring.datasource.url=${POSTGRES_URL:jdbc:postgresql://localhost/petclinic}
+-spring.datasource.username=${POSTGRES_USER:petclinic}
+-spring.datasource.password=${POSTGRES_PASS:petclinic}
+-# SQL is written to be idempotent so this is safe
+-spring.datasource.initialization-mode=always
```

Comment:
why did we drop these lines? If I understand correctly here: https://docs.spring.io/spring-boot/docs/2.1.0.M1/reference/html/howto-database-initialization.html#howto-database-initialization 

> Spring Boot automatically creates the schema of an embedded DataSource. This behaviour can be customized by using the spring.datasource.initialization-mode property. For instance, if you want to always initialize the DataSource regardless of its type:
>
> spring.datasource.initialization-mode=always

Is this something what we want actually to happen and with that we can drop that initiContainer?